task1/js/calc.js

```diff
@@ -0,0 +1,46 @@
+﻿function addition(a, b) {
+  return a + b;
+}
+
+function subtraction(a, b) {
+  return a - b;
+}
+
+function multiplication(a, b) {
+  return a * b;
+}
+
+function division(a, b) {
+  return a / b;
+}
+
+function integerDivision(a, b) {
+  return parseInt(a / b, 10);
```

Comment:
I'm ok with `parseInt`, but you could also find Math.trunc or polyfil `(a - a % b) / b`